src/Actions/Signup.php

```diff
@@ -147,10 +147,11 @@ private static function generate_username( $args ) {
 		if ( isset( $args['display_name'] ) ) {
 			$username = $args['display_name'];
 		} else {
-			$username =
+			$username = trim(
```

Comment:
This change comes from #7 
I incorrectly merged into this branch instead of master.